mp4parse_capi/src/lib.rs

```diff
@@ -429,22 +430,18 @@ pub unsafe extern fn mp4parse_get_track_info(parser: *mut mp4parse_parser, track
                 mp4parse_codec::MP3,
             AudioCodecSpecific::ES_Descriptor(_) =>
                 mp4parse_codec::UNKNOWN,
-            AudioCodecSpecific::MP3 =>
-                mp4parse_codec::MP3,
-            AudioCodecSpecific::AC3SpecificBox =>
-                mp4parse_codec::AC3,
-            AudioCodecSpecific::EC3SpecificBox =>
-                mp4parse_codec::EC3,
+            AudioCodecSpecific::MP3 | AudioCodecSpecific::AC3SpecificBox | AudioCodecSpecific::EC3SpecificBox =>
+                mp4parse_codec::UNKNOWN,
         },
         Some(SampleEntry::Video(ref video)) => match video.codec_specific {
             VideoCodecSpecific::VPxConfig(_) =>
                 mp4parse_codec::VP9,
             VideoCodecSpecific::AVCConfig(_) =>
                 mp4parse_codec::AVC,
-            VideoCodecSpecific::ESDSConfig(_) =>
-                mp4parse_codec::MP4V,
+            VideoCodecSpecific::ESDSConfig(_) => // Gecko doesn't support MP4V (14496-2) video.
+                mp4parse_codec::UNKNOWN,
             VideoCodecSpecific::JPEG =>
```

Comment:
Is it worth keeping the internal parsing code and enum entries for JPEG, AC3/EC3 at this point?